Test test_fill_parking_regions_mgmt_cmd fails in Docker environment

**Bug description**
Test ```test_fill_parking_regions_mgmt_cmd``` in ```parkings/tests/test_fill_parking_regions_command.py``` fails in Docker environment.

**To Reproduce**
Steps to reproduce the behavior:
1. Set up your .env file if you don't have one yet (e.g. by running ```echo 'DEBUG=True' > .env``` in the checkout's root)
2. Run ```docker-compose up --build```
3. Run ```docker-compose exec api pytest parkings/tests/test_fill_parking_regions_command.py```

Example output:
```python
================================================= test session starts ==================================================
platform linux -- Python 3.7.12, pytest-6.0.1, py-1.9.0, pluggy-0.13.1
django: settings: parkkihubi.settings_test (from ini)
rootdir: /app, configfile: setup.cfg
plugins: Faker-4.1.2, django-3.9.0, cov-2.10.1, requests-mock-1.8.0, factoryboy-2.0.3
collected 1 item

parkings/tests/test_fill_parking_regions_command.py F                                                            [100%]

======================================================= FAILURES =======================================================
__________________________________________ test_fill_parking_regions_mgmt_cmd __________________________________________
    @pytest.mark.django_db
    def test_fill_parking_regions_mgmt_cmd():
>       (parkings, regions) = create_parkings_and_regions()

parkings/tests/test_fill_parking_regions_command.py:15:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
parking_count = 100, region_count = 20

    def create_parkings_and_regions(parking_count=100, region_count=20):
        domain = EnforcementDomain.get_default_domain()
        regions = RegionFactory.create_batch(region_count, domain=domain)
        parkings = ParkingFactory.create_batch(parking_count, domain=domain)

        centroids = [region.geom.centroid for region in regions]
        touching_points = [p for p in centroids if intersects_with_any(p, regions)]

        # Make sure that some of the parkings are inside the regions
        for (point, parking) in zip(touching_points, parkings):
            parking.location = point
            parking.save()

        for parking in parkings:  # pragma: no cover
            if intersects_with_any(parking.location, regions):
>               assert parking.region
E               AssertionError

parkings/tests/utils.py:50: AssertionError
=============================================== short test summary info ================================================
FAILED parkings/tests/test_fill_parking_regions_command.py::test_fill_parking_regions_mgmt_cmd - AssertionError
================================================== 1 failed in 8.46s ===================================================
```

**Expected behavior**
Test ```test_fill_parking_regions_mgmt_cmd``` in ```parkings/tests/test_fill_parking_regions_command.py``` should pass in Docker environment.

**Additional context**
Ran the pytest in docker environment under Windows 10 64-bit and Docker Desktop 4.1.1 (69879).
The used code version was 88a8ecf0dc595df3a60d7e88efcdc5bc246ac2b1.

Docker container uses **Debian GNU/Linux 11** (bullseye):
```
docker-compose exec api bash -c "cat /etc/os-release | grep PRETTY_NAME"
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
```

Docker container uses **GDAL 3.2.2**:
```
docker-compose exec api gdalinfo --version
GDAL 3.2.2, released 2021/03/05
```

Docker container uses **Django 2.2.15** because requirements.txt contains "django==2.2.15".

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM python:3.7-slim as appbase
+FROM python:3.7-slim-buster as appbase
 
 ENV PYTHONBUFFERED 1
 

